src/app/components/book/book.component.html

```diff
@@ -0,0 +1,6 @@
+<div class="card">
+  <img [src]="book.image_url" alt="Book cover" class="book-cover">
+  <p class="title">{{book.title}}</p>
+  <p class="author">{{book.author}}</p>
+  <button class="btn-add">+</button>
```

Comment:
And the on click event?

Reply:
the on click event is added to this button in the following branch